common/src/main/java/com/revenuecat/purchases/common/ReceiptInfo.kt

```diff
@@ -3,7 +3,7 @@ package com.revenuecat.purchases.common
 import com.revenuecat.purchases.models.ProductDetails
 
 class ReceiptInfo(
-    val productID: String,
+    val productIDs: List<String>,
```

Comment:
i might just not be remembering correctly, but how can we have multiple product IDs but only one product details object?

Reply:
The main reason why we are adding this new parameter to the call is so we can know when this goes live. There are still some questions regarding how this is going to work on the Google side, some of them change our data model in the backend. For now we just want to know if there ever is a transaction with more than one product ID to know when this is live.

There is going to be more future work around this, like adding the price, currency or durations of all products.

Reply:
got it. perhaps we should outline that in the release notes, so people don't expect full functionality right away